examples/fluids/navierstokes.c

```diff
@@ -205,7 +211,7 @@ int main(int argc, char **argv) {
   // ---------------------------------------------------------------------------
   // Print problem summary
   // ---------------------------------------------------------------------------
-  if (!app_ctx->test_mode) {
+  if (!app_ctx->test_type) {
```

Comment:
minor - in other places you have
```suggestion
  if (app_ctx->test_type != TESTTYPE_NONE) {
```

Reply:
Good catch!